astropy/io/ascii/tests/data/cds/null/ReadMe1

```diff
@@ -60,7 +60,7 @@ Byte-by-byte Description of file: table.dat
   26- 30  F5.2  eV        chiEx     [1/20843]?+= Catalogue Identification Number
   32- 37  F6.2  ---       loggf     ]0/140[?Temperature class codified (10)
   39- 43  F5.1  0.1pm     EW        ?=-9.9 Equivalent width (in mA)
-  44- 45  F5.1  0.1pm     EW        [1/52]?-= Equivalent width (in mA)
+  44- 45  F5.1  0.1pm     EW_1      [1/52]?-= Equivalent width (in mA)
   46- 49  F4.1  0.1pm   e_EW        [1/6]?+ Luminosity class codified (11)
   51- 56  F6.3  ---       Q         ?=-9.999 DAOSPEC quality parameter Q
```

Comment:
```suggestion
  44- 48  F5.1  0.1pm     EW_1      [1/52]?-= Another equivalent width (in mA)
  49- 52  F4.1  0.1pm   e_EW        [1/6]?+ Luminosity class codified (11)
  53- 59  F6.3  ---       Q         ?=-9.999 DAOSPEC quality parameter Q
```
I found now that the assigning of unique column names is automatically done in https://github.com/astropy/astropy/blob/651ea27844801b19895b87e6ec7f776b1fd39a61/astropy/io/ascii/core.py#L1292-L1294
so this is not specific to CDS and there is probably no need to test the duplicate label here.
But I'd still vote for fixing the format specification (this would require a new `table1.dat` that actually has the extra columns and then might actually include some data so the _invalid input for `float`_ would not occur in the first place...)
Alternatively could perhaps change the format to `F2.0`, but that does not sound very sensible either.

Reply:
These changes still break the tests.

```
test_cds_no_whitespace - ValueError: Column e_EW failed to convert: could not convert string to float: '2  0'
test_cds_order - ValueError: Column e_EW failed to convert: could not convert string to float: '2  0'
```

Reply:
See comment above, you will need to create a new `table1.dat` with a matching data section, e.g. insert something in positions 46-48 like
```
Cr110   2108 6696.79 Al1  4.02  -1.42  29.5 -8.5 2.2  0.289
```
(I'd suggest to use valid numbers, although if you still wish to test the `fill_values` one row could be left empty).